tempodb/backend/raw.go

```diff
@@ -129,7 +131,17 @@ func (r *reader) ReadRange(ctx context.Context, name string, blockID uuid.UUID,
 }
 
 func (r *reader) Tenants(ctx context.Context) ([]string, error) {
-	return r.r.List(ctx, nil)
+	list, err := r.r.List(ctx, nil)
+
+	// this filter is added to fix a GCS usage stats issue that would result in ""
+	var filteredList []string
```

Comment:
@zalegrala @ie-pham 

minor but we should have preallocated this slice to match the tenants length.

a test on this behavior also would have been nice.